APE0.rst

```diff
@@ -0,0 +1,360 @@
+* Phase 2, Voting: Each Voting Member can vote for zero or more of the
+  candidates, up to the number of candidates. Voting is performed anonymously.
+  Candidates are ranked by the total number of votes they receive. If a tie
+  occurs, it may be resolved by mutual agreement among the candidates, or else
+  the winner will be chosen at random.
+
+The election process is managed by a non-voting Returns Officer nominated by the
+outgoing Coordination Committee. For the initial election, the Returns Officer
+will be nominated by the NumFOCUS Executive officer or their delegate. Elections
+will be carried out every year, provided that at least one of the Coordination
+Committee members has reached the end of their term or decided to step down.
+
+Term
+^^^^
+Each Coordination Committee member's term runs for three years from when
+the election results are finalized, except when the election was to fill a
+vacancy created partway through a term. In that case, the newly-elected member’s
+term runs for the remainder of the term of the person they are replacing. There
+is no limit to the number of terms that a single individual can be elected for.
+
+For the initial election of Coordination Committee members, two seats will have
+terms of one year, one has a term of two years, and one has a term of three
+years to create a staggered set of replacements and provide continuity in the
+Coordination Committee.
+
+Vacancies
+^^^^^^^^^
+Coordination Committee members may resign their position at any time.
+
+Whenever there is a vacancy during the regular Coordination Committee term an
+election must take place.
+
+If a Coordination Committee member cannot be contacted for longer than two
+months without prior notification (for example, due to planned leave), then the
+rest of the Coordination Committee may vote to trigger an election to replace
+them for the duration of their term.
+
+Removing Members
+^^^^^^^^^^^^^^^^
+In exceptional circumstances, the Voting Members may remove one or more sitting
+Coordination Committee members via a single vote.
+
+Such a vote is triggered when a Voting Member calls for one publicly on an
+appropriate Project-wide communication channel, and two other active Voting
+Members second the proposal.
+
+For Coordination Committee Member(s) to be removed, two-thirds of the active
+Voting Members must vote in support of the removal.
+
+To avoid repeated removal votes, any individual Voting Member who has called
+for, or seconded such a vote, may not call for or second a vote to remove any
+other Coordination Committee member for one year from the original motion.
+
+The Ombudsperson
+----------------
+The Ombudsperson represents the interests of the Astropy community by providing
+an alternative point of contact for sensitive issues such as code of conduct
+violations and ethical concerns. Candidates for this project role are publicly
+nominated by the Coordination Committee, after which the Coordination Committee
```

Comment:
Since we're now lacking an Ombudsperson, does that mean we have to wait for the new CoCo to form first via election before one can be appointed?

Reply:
The thinking right now is that *before* this doc comes into force the old "rules" (unwritten) still apply, which means we can put someone in the role.  Fortunately we have a candidate which I'll be sending a note about later today, so I think this will be in place *before* the election (fortunately).